presto-main/src/main/java/io/prestosql/sql/planner/QueryPlanner.java

```diff
@@ -484,130 +347,68 @@ private PlanBuilder aggregate(PlanBuilder subPlan, QuerySpecification node)
                 .map(OrderBy::getSortItems)
```

Comment:
this is not related to this change, but as a refactor

```
.filter(Optional::isPresent)
.map(Optional::get)
.map(OrderBy::getSortItems)
```
could be replaced with `.map(NodeUtils::getSortItemsFromOrderBy)`